### PR TITLE
Add notes about our reaction to CircleCI security incident

### PR DESCRIPTION
**Pull request by bkabrda**
### What does this PR do?
Adds notes about our reaction to CircleCI security incident

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+On January 4th, 2023, we were notified by CircleCI that they were investigating a [security incident](https://circleci.com/blog/january-4-2023-security-alert/) that may have led to leaking stored secrets. We identified a single secret stored in CircleCI that could theoretically be misused by a potential attacker, an old RPM GNU Privacy Guard (GPG) private signing key and its passphrase. This page documents the implications of the potential leak and the measures we are taking to mitigate any risks to our customers.
+
+<div class="alert alert-info">
+<strong>Note</strong>: As of January 10th, 2023, we have no indication that the key was actually leaked or misused, but we are still taking the following actions out of an abundance of caution.
+</div>
+
+## The Affected Key
+
+The RPM GPG signing key has fingerprint `4172A230`, and is accessible in [our signing keys location](https://keys.datadoghq.com/DATADOG_RPM_KEY.public). This key was historically used to sign Agent 5 releases and Agent 6 releases up to (and including) 6.13.0.
```
Comment (KSerrania) [inline content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md:14]: Something I just thought about (can be addressed later): should we use the full fingerprint of the key, to be in line with what we do in other docs pages (like https://docs.datadoghq.com/data_security/agent/)?
```suggestion
The RPM GPG signing key has fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location](https://keys.datadoghq.com/DATADOG_RPM_KEY.public). This key was historically used to sign Agent 5 releases and Agent 6 releases up to (and including) 6.13.0.
```

File: content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+## Implications of no Longer Trusting the Affected Key
+
+* If your system uses Agent 7, there is no implication. Agent 7 packages were never signed with the affected key.
+* Your system will no longer be able to install Agent 6 < 6.14.0. We recommend upgrading to Agent 6 >= 6.14.0 or Agent 7.
+* If your system uses Agent 5, you will no longer be able to install any currently released Agent 5 version.
+
+## What We're Doing to Mitigate the Implications
+
+* We're working towards releasing a new Agent 5 version signed with the [current RPM signing key](https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public), `FD4BF915`.
```
Comment (KSerrania) [inline content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md:101]: Nit (same as above):
```suggestion
* We're working towards releasing a new Agent 5 version signed with the [current RPM signing key](https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public), `C6559B690CA882F023BDF3F63F4D1729FD4BF915`.
```

**Review by KSerrania — APPROVED** [review]

File: content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+On January 4th, 2023, Datadog was notified by CircleCI that they were investigating a [security incident][1] that may have led to leaking stored secrets. Datadog identified a single secret stored in CircleCI that could theoretically be misused by a potential attacker, an old RPM GNU Privacy Guard (GPG) private signing key and its passphrase. This page documents the implications of the potential leak and the measures Datadog is taking to mitigate any risks to our customers.
+
+<div class="alert alert-info">
+<strong>Note</strong>: As of January 10th, 2023, Datadog has no indication that the key was actually leaked or misused, but we are still taking the following actions out of an abundance of caution.
+</div>
+
+## The affected key
+
+The RPM GPG signing key has fingerprint `4172A230`, and is accessible in [our signing keys location][2]. This key was historically used to sign Agent 5 releases and Agent 6 releases up to (and including) 6.13.0.
```
Comment (apigirl) [inline content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md:14]:
```suggestion
The RPM GPG signing key has fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location][2]. This key was historically used to sign Agent 5 releases and Agent 6 releases up to (and including) 6.13.0.
```
this is @KSerranias suggestion that got overwritten

File: content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+```bash
+$ sudo rpm --erase gpg-pubkey-4172a230-55dd14f6
+```
+
+To delete the key from the Datadog repofile, remove the `gpgkey` line that ends with `DATADOG_RPM_KEY.public`. If this was the only `gpgkey` entry in your repofile, replace it with `https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public`. Note that this has implications explained in the section [Implications of no Longer Trusting the Affected Key](#implications-of-no-longer-trusting-the-affected-key).
+
+### Usage of automation tools
+
+Usage of automation tools like the Datadog Ansible role (see the full list in [What we're doing to mitigate the implications](#what-were-doing-to-mitigate-the-implications)) might reverse the manual changes recommended above. Until we release new versions that fix this, we recommend adding these manual changes to your automation tool runbooks.
```
Comment (bkabrda) [inline content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md:75]:
```suggestion
Usage of automation tools like the Datadog Ansible role (see the full list in [What we're doing to mitigate the implications](#what-datadog-is-doing-to-mitigate-the-implications)) might reverse the manual changes recommended above. Until we release new versions that fix this, we recommend adding these manual changes to your automation tool runbooks.
```

File: content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+$ curl -o /tmp/rpm_check.sh https://docs.datadoghq.com/resources/sh/rpm_check.sh && chmod +x /tmp/rpm_check.sh
+$ /tmp/rpm_check.sh
+```
+
+The script will:
+
+* Verify that any installed Datadog packages signed by the affected key were indeed built by Datadog by verifying the full GPG signature of RPM headers and payload.
+* Search for any packages signed by the affected key that weren't built by Datadog.
+
+Lines starting with `[ ERROR ]` are considered suspicious and should be reported to [Datadog Support][4] along with the full script output.
```
Comment (bkabrda) [inline content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md:91]:
```suggestion
Lines starting with `[ ERROR ]` should be reported to [Datadog Support][4] along with the full script output.
```

**Review by apigirl — APPROVED** [review]

File: content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+## Implications of no longer trusting the affected key
+
+* If your system uses Agent 7, there is no implication. Agent 7 packages were never signed with the affected key.
+* Your system will no longer be able to install Agent 6 < 6.14.0. We recommend upgrading to Agent 6 >= 6.14.0 or Agent 7.
+* If your system uses Agent 5, you will no longer be able to install any currently released Agent 5 version.
+
+## What Datadog is doing to mitigate the implications
+
+* We're working towards releasing a new Agent 5 version signed with the [current RPM signing key](https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public), `C6559B690CA882F023BDF3F63F4D1729FD4BF915`.
```
Comment (olivielpeau) [inline content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md:101]:
```suggestion
* We're working towards releasing a new Agent 5 version for CentOS/RHEL signed with the [current RPM signing key](https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public), `C6559B690CA882F023BDF3F63F4D1729FD4BF915`.
```

**Pull request merged**